script/input/1/base.json

```diff
@@ -0,0 +1,5 @@
+{
+    "cctpTokenMessenger": "0x1682Ae6375C4E4A97e4B583BC394c861A46D8962",
+    "safe": "0x22fB6fe2B9aA289D26724eCBD5a679751A4508b5",
```

Comment:
Staging safe on base is `0xaB959A6F88b8D966c44a7cDC2049Ba9669EBf047`

Reply:
Updated

Reply:
Updated